### DISCUSSION TITLE
Is there a way to prevent SRW v2.1.0 from writing 'qos=None' in the SBATCH directives?

### DISCUSSION THREAD

**Discussion by padhrigmccarthy** [ufs-community/ufs-srweather-app#517] [Q&A]
Hi- 

I'm trying to transition from a develop branch deployment of SRW (from Aug/Sept 2022) to using v2.1.0. I'm running on an HPC at Boise State University, so there's no pre-configuration. I've created all the necessary wflow_host.lua, build_host_intel.lua, and machine/host.yaml files. Now that I've switched to v2.1.0, I'm not able to get the model to run because of errors like this one:

`12/13/22 20:24:58 GMT :: FV3LAM_wflow.xml :: Submission of make_grid failed!  sbatch: error: Batch job submission failed: Invalid qos specification`

The submission looks like this (from log.launch_FV3LAM_wflow):

```
#! /bin/sh
#SBATCH --account=pmccarthy
#SBATCH --qos=None
#SBATCH --partition=ncar
#SBATCH --nodes=1-1
#SBATCH --tasks-per-node=1
#SBATCH -t 00:45:00
#SBATCH --job-name=get_extrn_ics
#SBATCH -o /ipcscratch/pmccarthy/SRW_build/expt_dirs/3km_CUSTOM_RRFSv1beta/log/get_extrn_ics_2019061518.log
#SBATCH
#SBATCH --comment=761c491fcad52da408617c63f666cce3
export GLOBAL_VAR_DEFNS_FP='/ipcscratch/pmccarthy/SRW_build/expt_dirs/3km_CUSTOM_RRFSv1beta/var_defns.sh'
export PDY='20190615'
export cyc='18'
export subcyc='00'
export LOGDIR='/ipcscratch/pmccarthy/SRW_build/expt_dirs/3km_CUSTOM_RRFSv1beta/log'
export ICS_OR_LBCS='ICS'
/ipcscratch/pmccarthy/SRW_build/ufs-srweather-app/ush/load_modules_run_task.sh "get_extrn_ics" "/ipcscratch/pmccarthy/SRW_build/ufs-srweather-app/jobs/JREGIONAL_GET_EXTRN_MDL_FILES"
```

 

The problem is that this cluster doesn't accept `--qos=something` directives because there are no queues configured.

Is there a way to write '--qos='   or better yet, to leave the directive off altogether? I can't seem to get SRW to submit jobs with anything other than '--qos=something' or '--qos=None'.

Thank you, -Paddy McCarthy.

**Comment by mark-a-potts** [comment]
Hi Paddy. If you are using rocoto to run the workflow, I think you may need to look in ufs-srweather-app/parm/FV3LAM_wflow.xml on lines 140-152. You should remove the \<queue\>.....\</queue\> sections from each line.

If you are using the wrapper scripts, you can just comment out the qos line in ufs-srweather-app/ush/wrappers/sq_job.sh script.

**Reply by mark-a-potts** [reply]
Just realized that the xml sections I entered above didn't come through initially. Updated to make sure the "queue" part shows up.
